objectivec/GPBWellKnownTypes.h

```diff
@@ -49,4 +51,14 @@ NS_ASSUME_NONNULL_BEGIN
 - (instancetype)initWithTimeIntervalSince1970:(NSTimeInterval)timeIntervalSince1970;
 @end
 
+// Extension to GPBAny to support packFrom and unpackTo for abitriary message.
+@interface GPBAny (GPBWellKnownTypes)
+- (void)unpackTo:(GPBMessage*)message;
+- (void)packFrom:(GPBMessage*)message;
+- (BOOL)is:(Class)messageType;
```

Comment:
NSObject already has this at the root level as isKindOf: and isMemberOf:
There is no need to add "is".